ci-operator/step-registry/hypershift/kubevirt/create/hypershift-kubevirt-create-chain.yaml

```diff
@@ -18,12 +37,12 @@ chain:
 
       CLUSTER_NAMESPACE=clusters-${CLUSTER_NAME}
 
-      echo "$(date) Creating HyperShift cluster ${CLUSTER_NAME}"
+      echo "$(date) Creating HyperShift guest cluster ${CLUSTER_NAME}"
       /usr/bin/hypershift create cluster kubevirt \
         --name ${CLUSTER_NAME} \
         --node-pool-replicas ${HYPERSHIFT_NODE_COUNT} \
-        --memory 16Gi \
-        --cores 4 \
+        --memory ${HYPERSHIFT_NODE_MEMORY}Gi \
+        --cores ${HYPERSHIFT_NODE_CPU_CORES} \
```

Comment:
I believe we'll want to optionally set the kubevirt csi storage class mappings here.

We could trigger this with an env var. KUBEVIRT_CSI_INFRA_SC=gp3-csi

```
EXTRA_ARGS=""

if [ -n "$KUBEVIRT_CSI_INFRA" ]; then
    EXTRA_ARGS="$EXTRA_ARGS --infra-storage-class-mapping=$KUBEVIRT_CSI_INFRA/guest-sc"
fi
```

Then add EXTRA_ARGS to the create command


AFter that, once the hosted cluster is completely online, We'll want to mark that `guest-sc` as the default storage class.

```oc --kubeconfig=${SHARED_DIR}/nested_kubeconfig annotate --overwrite sc guest-sc storageclass.kubernetes.io/is-default-class='true'```